operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml

```diff
@@ -990,6 +990,7 @@ spec:
                       Reducing the number of threads may cause the ingress controller
                       to perform poorly."
                     format: int32
+                    maximum: 64
```

Comment:
Maybe we should qualify why 64 is a max (i.e., it's not arbitrary, but imposed by haproxy-2.2.x).

Reply:
That's a good idea. I'll update the description.